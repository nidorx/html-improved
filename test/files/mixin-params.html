<mixin name="article" params="title, authorName, class-name, complexObject">
    <div class="article !{className}">
        <div class="article-wrapper">
            <h1>#{title}</h1>
            <span>#{authorName}</span>
            <span>!{complexObject.id}</span>
            <span>!{complexObject.contents[0].name}</span>
            <span>!{complexObject.contents[1].name}</span>
        </div>
    </div>
</mixin>

<article
        title="Hello world"
        author-name="John Doe"
        class-name="figure-3d"
        complex-object="{
            'id':'features',
            'contents': [
                {
                    'name':'John'
                },
                {
                    'name':'Alex'
                }
            ]
        }"
/>
