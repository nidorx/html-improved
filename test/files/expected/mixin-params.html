<div class="article figure-3d">
    <div class="article-wrapper">
        <h1>Hello world</h1>
        <span>John Doe</span>
        <span>features</span>
        <span>John</span>
        <span>Alex</span>
    </div>
</div>
